### PR TITLE
create header

### PR DESCRIPTION
**Pull request by Maichonok**
I developed Header component according to my design in Figma.

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import "./Header.css";
+import Girl from "../../assets/girl.svg";
+import Baikal from "../../assets/Baikal.svg";
+import GirlAndBoy from "../../assets/girl_and_boy.svg";
+
```
Comment (kurumkan) [inline src/components/Header/Header.tsx:6]: https://create-react-app.dev/docs/importing-a-component/#absolute-imports

Comment (Maichonok) [reply src/components/Header/Header.tsx:6]: Thank you! :)

File: src/App.tsx [hunk 2]
```diff
@@ -1,9 +1,9 @@
-import Home from "../src/pages/Home";
+import Header from "./components/Header/Header";
```
Comment (kurumkan) [inline src/App.tsx:2]: please fix the path here and all across the project

Comment (Maichonok) [reply src/App.tsx:2]: Okay ;)

**Pull request merged**